docs/devel/infrastructureguide.rst

```diff
@@ -60,7 +60,7 @@ The following services are used to track code quality:
 
 * `Codacy <https://app.codacy.com/project/Unidata/MetPy/dashboard>`_
 * `Code Climate <https://codeclimate.com/github/Unidata/MetPy>`_
-* `Scrutinizer <https://scrutinizer-ci.com/g/Unidata/MetPy/?branch=master)>`_
+* `Scrutinizer <https://scrutinizer-ci.com/g/Unidata/MetPy/?branch=main)>`_
```

Comment:
I don't think we're exactly _using_ Scrutinizer, heh. Drop this?

Reply:
Yeah, it's hooked up but...yeah.